source/likely.js

```diff
@@ -2,5 +2,6 @@
 
 var likely = require('./index.js');
 
-window.likely = likely;
 window.addEventListener('load', likely.initiate);
```

Comment:
> take your time and feel free to return to this when it’s OK for you, or I’ll do this on my own earlier if I find some time myself.

@andrew--r @iamakulov Guys! Does anybody come back to the task? If everyone's busy I can take it.